app/src/main/java/com/example/mymovie/MainActivity.kt

```diff
@@ -3,20 +3,29 @@ package com.example.mymovie
 import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.view.View
+import android.widget.ImageView
 import android.widget.TextView
+import androidx.core.content.ContextCompat.startActivity
+
+//  Add FragmentMoviesList to the MainActivity
+//  create the transaction with transition of navigation to FragmentMoviesDetails by click on Card with Film
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
-        val textView: TextView = findViewById(R.id.first_text_view)
-        textView.setOnClickListener { moveToNextScreen() }
+        supportFragmentManager.beginTransaction()
```

Comment:
after opening  MainActivity rotate screen few times
and press back few time

this should not happed